src/components/claiming/UploadFile.vue

```diff
@@ -21,19 +21,25 @@
         :http-request="handleUpload"
         :on-change="handleUpload"
         :limit="1"
+        :file-list="file"
```

Comment:
Is there a way to change the UI when the file is uploaded?

Reply:
It already changes. When the file is uploaded a green tick is shown to the user.

Reply:
Discuss in Issue #980